### ISSUE TITLE
ARIA: Editorial cleanup needed to "Presentational Roles Conflict Resolution" section

### ISSUE THREAD

**Issue by joanmarie** [w3c/aria#646]
Spinning this issue off from https://github.com/w3c/aria/issues/492#issuecomment-329386912, in which @mcking65 states:

> And this paragraph needs a rewrite to make it intelligible:
>
>> Host languages elements, having implicit presentational roles for which no roles, may be applied, must never be exposed to in the accessibility tree. With this exception, user agents must always expose global WAI-ARIA states and properties to accessibility APIs. In this case, the user agent ignores the presentation role and exposes the element according to its implicit native semantics. However, user agents must ignore any non-global, role-specific WAI-ARIA states and properties, unless it is on an inherited presentational role where an explicit role is applied.
>
> The only way I came to understand it was because of the subsequent example.

**Comment by mcking65** [comment]
@joanmarie, since this is editorial, can we resolve it for 1.1? I see you have the 1.1 label on it.

**Comment by joanmarie** [comment]
That's the plan @mcking65, if it's truly editorial. And from you'll pull request it does indeed seem to be. :smile: But as I commented just now in my PR review, I think there's a tiny flow issue in the wording. If you have another spare moment or two to tweak that, it would be great.

**Comment by joanmarie** [comment]
I was hoping that we'd get to this during the 1.1 cycle. But getting all 1.1-related implementations in place and then verifying them through testing wound up occupying wound up occupying the bulk of everyone's time. Given that we're at a "pencils down" point in our progression along the Rec track, that the existing text and/or the example associated with it seem to make things sufficiently understandable to implementors, and that we can address authors needs through the Practices document, let's postpone this to 1.2.

**Comment by carmacleod** [comment]
Actually, reading this section again, I see that even more clarity is needed... so in PR #852 I've reworded the points and created a list. 

@mcking65 and @joanmarie I've asked you both to review. Note that the diff is hard to follow, so I think the easiest way to review this is to read the [preview](https://pr-preview.s3.amazonaws.com/w3c/aria/pull/852.html#conflict_resolution_presentation_none) from the start of the Presentational Roles Conflict Resolution section down to (but not including) the Characteristics table (about 4 paragraphs, including list and example).
